src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -229,7 +225,7 @@ namespace ChillerReformulatedEIR {
                                                 this->CondOutletNodeNum,
                                                 this->QCondenser,
                                                 this->CondInletTemp,
-                                                RefEIRReport(this->myIdx).CondOutletTemp,
+                                                this->CondOutletTemp,
```

Comment:
It looks like there are a couple things going on that could cause the diff.  Is it due to the removal of the RefEIRReport instance?  Or the module level variable?  Are you able to split out which is the root cause?  It could make it even easier to digest the diff and verify we understand the implications.  I feel like in the end it's going to be found that the root problem was that a temp was not properly updated on every `return` path, but I'd like to see if we can nail down the exact line or code path that is causing it.  It may even help resolve #7680